app/src/main/res/values-de/strings.xml

```diff
@@ -52,6 +52,8 @@
     <string name="aiButtonInsert">Einfügen</string>
     <string name="aiButtonRefine">Verfeinern</string>
     <string name="aiButtonRetry">Wiederholen</string>
+    <string name="aiDiscoveryDescription">Vereinfachen Sie das Schreiben von Nachrichten mit künstlicher Intelligenz.</string>
+    <string name="aiDiscoveryTitle">Schreiben Sie Ihre E-Mails mit AI</string>
```

Comment:
A bit strange to use AI then künstlicher Intelligenz
If we want to use AI, we should probably use artifizielle Intelligenz
Otherwise, maybe use KI

Reply:
At infomaniak we use "Künstliche Intelligenz", so yes "KI".